dbms/programs/obfuscator/Obfuscator.cpp

```diff
@@ -943,159 +943,170 @@ class Obfuscator
 
 }
 
+#pragma GCC diagnostic ignored "-Wmissing-declarations"
+#pragma GCC diagnostic ignored "-Wunused-function"
 
 int mainEntryClickHouseObfuscator(int argc, char ** argv)
-try
```

Comment:
Function try block is obscure feature but why someone compains?

Reply:
It was me, I will revert it